packages/inline/src/components/v-text.ts

```diff
@@ -15,11 +15,18 @@ export class VText extends LitElement {
     'white-space': 'break-spaces',
   };
 
+  private parser = new DOMParser();
+
   override render() {
     // we need to avoid \n appearing before and after the span element, which will
     // cause the sync problem about the cursor position
+    const parsed = this.parser.parseFromString(
+      this.str.replace(/\s/g, '&nbsp;'),
```

Comment:
Seems that this breaks the test cases that depends on the rendered result. To land this, corresponding test cases also needs be updated.

Reply:
Looks like yes, I will update this PR.